drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/GuidedDecisionTableGeneratorListenerTest.java

```diff
@@ -179,45 +180,45 @@ public void testAttributes() {
 
         //Check individual attributes
         AttributeCol52 attrCol2 = ((AttributeCol52) columns.get(2));
-        assertEquals(GuidedDecisionTable52.SALIENCE_ATTR,
+        assertEquals(Attribute.SALIENCE,
```

Comment:
@jomarko Tests are failing in this point, this assertion compares 'Attribute.SALIENCE.toString()' = "SALIENCE" with 
"salience". You should change 'Attribute.SALIENCE' with 'Attribute.SALIENCE.getAttribute()` to fix it. Hope this helps!

Reply:
@yesamer yes, I am sorry, I haven't checked all tests properly. I hope now all of them are fixed, see the last commit.